### ISSUE TITLE
Add a separate scylla node as the manager's backend

### ISSUE THREAD

**Issue by ShlomiBalalis** [scylladb/scylla-ccm#162]
Currently, the scylla manager uses the same cluster that is used for testing as a backend. This creates a problematic situation on certain test cases. Can't we create a separate scylla cluster that will be used as the backend?

Migrated to Jira: [DTEST-2](https://scylladb.atlassian.net/browse/DTEST-2)

[DTEST-2]: https://scylladb.atlassian.net/browse/DTEST-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by ShlomiBalalis** [comment]
@slivne

**Comment by fruch** [comment]
@mikliapko is that something of interest for you, or we can close it ?

**Comment by mikliapko** [comment]
I don't think we would invest time into that considering Manager dtests are mostly in support mode without any active test development there

**Comment by dani-tweig** [comment]
Closing this issue as it was moved to Jira.
Please continue the thread in https://scylladb.atlassian.net/browse/DTEST-2
